content/search/results.js

```diff
@@ -62,11 +74,14 @@
     var $input = document.getElementById('input-search');
     var q = parseQueryParams(window.location.search);
 
-    if (q.length)
+    if (q.length) {
       showResults($main, $input, q);
+      trackSearch(q);
+    }
 
-    $input.oninput = function (e) {
+    $input.addEventListener('input', debounce(function () {
       showResults($main, $input, $input.value);
-    };
+      trackSearch($input.value);
```

Comment:
did you intentionally debounce the showing of the search results, or did you just want to debounce the tracking?

Reply:
@dperrymorrow originally I wanted to only debounce the tracking, but then I would need 2 listeners. Would that be OK?